mix.exs

```diff
@@ -53,7 +54,8 @@ defmodule Xebow.MixProject do
       dialyzer: "do cmd mkdir -p _build/#{Mix.target()}_#{Mix.env()}/plt, dialyzer",
       "docs.show": "do docs, cmd xdg-open doc/index.html",
       loadconfig: [&bootstrap/1],
-      upload: "cmd ./upload.sh"
+      upload: "cmd ./upload.sh",
+      setup: ["deps.get", "cmd npm install --prefix assets"]
```

Comment:
:warning: (recommended)

It would be nice to alias `"cmd npm install --prefix assets"` and run it with `mix firmware` so the static assets compile into the firmware image.

Reply:
Previously captured in #58.

Reply:
>Also, it seems to be using a target of `host` when Mix gets the deps, so at least that task must be run with `MIX_TARGET=keybow` in the environment.

FYI we'll ideally want to target `host` for development of the UI because it'll be a faster development cycle to see the changes locally than to flash the hardware each time we want to test changes.

Captured in #46.